src/ui/field_formats_mixin.js

```diff
@@ -0,0 +1,7 @@
+import { FieldFormatsService } from '../core_plugins/kibana/common/field_formats/field_formats';
+import { getUiSettingDefaults } from '../core_plugins/kibana/ui_setting_defaults';
+
+export function fieldFormatsMixin(kbnServer, server) {
+  server.decorate('server', 'uiSettingDefaults', getUiSettingDefaults());
```

Comment:
This doesn't feel like it belongs here, specifically because it's related to uiSettings. I know you're using the uiSettingDefaults with the fieldFormats, but the `fieldFormatsMixin` really shouldn't be responsible for this.

Additionally, you should be able to do ` uiSettings.getDefaults();` and not have to expose this separately.